groupchannel-mention-members/src/main/java/com/sendbird/chat/sample/groupchannel/mentionmembers/ui/groupchannel/GroupChannelChatActivity.kt

```diff
@@ -415,7 +435,12 @@ class GroupChannelChatActivity : AppCompatActivity() {
         }
         val collection = messageCollection ?: return
         val channel = currentGroupChannel ?: return
+
         val params = UserMessageCreateParams().setMessage(message.trim())
+        val mentionedUsersIds = getMentionedUsersIdsFromMessage(message)
```

Comment:
This Feature wants to implement this [feature](https://sendbird.com/docs/chat/v4/android/guides/group-channel#2-mention-other-members-in-a-message), not the mention in the message content. 🥲

Reply:
Sorry! I think I misunderstood